src/docs/sanapi/available-metrics.md

```diff
@@ -82,21 +130,54 @@ you can fetch the list of available slugs with this query:
 
 **[Run in explorer](<https://api.santiment.net/graphiql?query=%7B%0A%20%20getMetric(metric%3A%20%22mvrv_usd%22)%7B%0A%20%20%20%20metadata%7B%0A%20%20%20%20%20%20availableSlugs%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D>)**
 
-There are 2 fields that can be queried on `getMetric` and each of them has additional subfields: `metadata` and `timeseriesData`
+There are 5 fields that can be queried on `getMetric`: `metadata`, `timeseriesData`, `histogramData`, `availableSince` and `lastDatetimeComputedAt`.
+Some of them has additional subfields: `metadata`, `timeseriesData` and `histogramData`
```

Comment:
Some of them have additional subfields: `metadata`, `timeseriesData` and `histogramData`